app/templates/ui/app/login/login.service.js

```diff
@@ -9,9 +9,10 @@
   function LoginService($http, $modal, $q, $rootScope, $state, $stateParams, messageBoardService) {
     var _loginMode = 'full'; // 'modal', 'top-right', or 'full'
     var _loginError;
-    var _isAuthenticated = false;
     var _toStateName;
     var _toStateParams;
+    var _isAuthenticated = isLoggedIn();
```

Comment:
Is it your intention to eval the isLoggedIn function right on the spot, or do you want to make isAuthenticated (way down) return a promise from that helper function? (it is doing first, but kinda guessing you want the latter?)